docs/basics/integrations/webhooks.md

```diff
@@ -71,3 +113,21 @@ Each event sends a POST request containing the job object with the following dat
+|---------------------|--------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `org_id`            | STRING | TRUE     | Id of build owner's Sauce Labs organization.                                                                                                                                          |
+| `user_id`           | STRING | TRUE     | Build owner's Sauce Labs user id.                                                                                                                                                     |
+| `team_ids`          | ARRAY  | TRUE     | A list of Sauce Labs team ids which the build owner belongs to.                                                                                                                       |
+| `creator_id`        | STRING | TRUE     | Id of build owner's Sauce Labs organization.                                                                                                                                          |
+| `build_id`          | STRING | TRUE     | Sauce Labs unique identifier of the build.                                                                                                                                            |
+| `build_name`        | STRING | TRUE     | The name that was given to the build.                                                                                                                                                 |
+| `build_url`         | STRING | TRUE     | Url with build details.                                                                                                                                                               |
+| `status`            | ENUM   | TRUE     | Status of the build. Possible values are:<br/><ul><li>`APPROVED`</li><li>`EMPTY`</li><li>`EQUAL`</li><li>`ERRORED`</li><li>`QUEUED`</li><li>`REJECTED`</li><li>`UNAPPROVED`</li></ul> |
+| `branch`            | STRING | FALSE    | The name of the branch.                                                                                                                                                               |
+| `project`           | STRING | FALSE    | The name of the project.                                                                                                                                                              |
+| `summary`           | JSON   | TRUE     | Contains the number of the snapshots of the following statuses: `errored`, `approved`, `equal`,  `rejected`, `unapproved`, `queued`                                                   |
+| `data_center`       | STRING | TRUE     | Data center when the build was executed.  Possible values are:<br/><ul><li>`us-west-1`</li><li>`eu-cental-1`</li><li>`us-east-1`</li></ul>                                            |
```

Comment:
Data center **where** the build was executed.

and `eu-cental-1` -> `eu-central-1`